lldb/source/Plugins/Process/elf-core/ProcessElfCore.h

```diff
@@ -117,6 +117,13 @@ class ProcessElfCore : public lldb_private::PostMortemProcess {
     lldb::addr_t end;
     lldb::addr_t file_ofs;
     std::string path;
+    lldb_private::UUID uuid; //.note.gnu.build-id
+  };
+
+  struct Section_Note {
+    uint32_t namesz;
+    uint32_t descsz;
+    uint32_t type;
```

Comment:
Move this to the ThreadElfCore.h header and add a 'parse' method to it, so the UUID is populated much like the PRPSINFO data and other useful info from the coredump.

Reply:
or move it into ELFHeader.h and then add a parse method. We are using a ton of stuff already from ELFHeader.h, so we should follow that path